net/http/src/THttpServer.cxx

```diff
@@ -317,7 +310,7 @@ void THttpServer::SetDefaultPage(const char *filename)
       fDefaultPage = fJSROOTSYS + "/files/online.htm";
 
    // force to read page content next time again
-   fDefaultPageCont.Clear();
+   fDefaultPageCont.clear();
```

Comment:
Wasn't `fDefaultPageCont` just created? Why does it need to be cleared here?

Reply:
No, it is not in constructor.